upgrade-cockroach.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Upgrade Your Cluster's Version of CockroachDB
+summary: Learn how to upgrade your CockroachDB cluster to a new version.
+toc: false
+toc_not_nested: true
+---
+
+Because of CockroachDB's multi-active availability design, you can perform a "rolling upgrade" of CockroachDB on your cluster. This means you can upgrade individual nodes in your cluster one at a time without bringing your cluster down at all.
+
+For rolling upgrades to work, though, your cluster must be behind a load balancer. If your application communicates with a single node, bringing it down to upgrade its CockroachDB binary, your application will not receive data, causing it to fail.
+
+## Manually Perform a Rolling Upgrade
+
+In advance of this procedure, you will need the IP addresses of all your cluster's nodes.
+
+1. Connect to the node.
+
+2. Stop `cockroach`:
+
+   ~~~ shell
+   $ cockroach quit [flags]
+   ~~~
+
+   `[flags]` includes the flags you [use to communicate with a node](stop-a-node.html), such as its `--host`.
+
+3. Rename the outdated `cockroach` binary:
+
+   ~~~ shell
+   $ i="$(which cockroach)"; mv "$i" "$i"_old
```

Comment:
This looks kind of magical, and it may not work as written (because the binary may not be on `$PATH`, or it may not be in a writeable directory). I think it's better in a server context to use the versioned (not `latest` tarballs) and leave the binaries in their versioned directories instead of copying them onto `$PATH`. (but this is one of the hazards of providing this kind of manual instruction - everyone has their own opinions about the right way to do things).